src/ui_interface.h

```diff
@@ -95,6 +97,12 @@ class CClientUIInterface
 
     /** New block has been accepted */
     boost::signals2::signal<void (const uint256& hash)> NotifyBlockTip;
+
+    /** New block has been accepted */
+    boost::signals2::signal<void (const CBlock& block)> NotifyAcceptBlock;
```

Comment:
As mentioned in one of the previous PRs, block acceptance is really not interesting or useful. It does not mean the block is part of the best chain, or even that it is valid. Use the NotifyBlockTip signal place, which indicates a new tip of the best chain was accepted.